bin/xbps-checkvers/main.c

```diff
@@ -200,9 +200,10 @@ show_usage(const char *prog)
 "  -I,--installed 		Check for outdated packages in rootdir, rather\n"
 "				than in the XBPS repositories.\n"
 "  -i,--ignore-conf-repos	Ignore repositories defined in xbps.d.\n"
+"  -m,--manual	Only process listed files.\n"
```

Comment:
Please fix the indent here.

Reply:
Fixed it while making the usage text match all the other xbps-* usage outputs.